docker-compose.yml

```diff
@@ -7,4 +7,4 @@ services:
     volumes:
       - .:/docs
     ports:
-      - 8000:8000
```

Comment:
I don't think that we want this to be changed

Reply:
Oh I had to change mine cause I had a service on that port already. I can change it back for the pr

Reply:
Once this change has been made I will approve and you can merge

Reply:
Ok I think i changed it back to the appropriate port of 8000